resources/data/nrf_connect_sdk-0.1.3.json

```diff
@@ -49,6 +49,62 @@
                 }
             ]
         },
+        {
+            "type": "Checkable",
+            "steps": [
+                {
+                    "type": "Step",
+                    "platforms": [
+                        { "platform": "darwin" }
+                    ],
+                    "title": "Installing additional tools for Connected Home over IP applications development",
+                    "description": [
+                        "_Note: If you are not interested in the [Connected Home over IP](https://www.connectedhomeip.com/) applications, you can ommit steps presented in this section._",
+                        "Download and extract [GN](https://gn.googlesource.com/gn/) build system executable archive used by the Connected Home over IP to generate output files:",
+                        {
+                            "type": "commands",
+                            "description": [
+                                "cd ~",
+                                "wget -O gn.zip https://chrome-infra-packages.appspot.com/dl/gn/gn/mac-amd64/+/latest",
+                                "unzip gn.zip gn",
+                                "rm gn.zip"
+                            ]
+                        },
+                        "Put GN location into the system PATH:",
+                        "```\necho \"export PATH=$PWD:$PATH\" >> $HOME/.bashrc\n```"
```

Comment:
Here we instruct to add the `$PWD` to $PATH, while we are not in `gn` folder, is that expected? Line 69.

Reply:
It is expected, as gn.zip is extracted straight to the `~`. I had a discussion with @Damian-Nordic about this and he noticed that gn is single executable file, so creating dedicated dictionary for one file may be pointless. Alternatively I can suggest replacing `cd ~` command with `mkdir ~/gn && cd ~/gn`. What is your opinion on this?

Reply:
Btw, I can see that in case of CMake the Getting Started Assistant is asking to modify `${HOME}/.zephyrrc` instead of `${HOME}/.bashrc` which is nice because it only modifies a user's PATH when he loads Zephyr build system. Also it will work even if someone is using another shell like `zsh`. How about using `.zephyrrc` here as well?